diff --git a/Tables/dbo.Winners.sql b/Tables/dbo.Winners.sql
index 814c1d2..bfb249e 100644
--- a/Tables/dbo.Winners.sql
+++ b/Tables/dbo.Winners.sql
@@ -3,7 +3,8 @@
 [Id] [int] NOT NULL,
 [ParticipantId] [int] NOT NULL,
 [Prize] [nvarchar] (50) COLLATE Polish_CI_AS NULL,
-[Test] [bit] NULL
+[Test] [bit] NULL,
+[Comments] [nvarchar] (100) COLLATE Polish_CI_AS NOT NULL
 ) ON [PRIMARY]
 GO
 ALTER TABLE [dbo].[Winners] ADD CONSTRAINT [PK_Winners] PRIMARY KEY CLUSTERED  ([Id]) ON [PRIMARY]
